package/com.unity.formats.usd/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 ### Bug Fixes
+- Fixed a bug where importing materials exported from a newer USD version would fail.
```

Comment:
I'd modify this to be more explicit i.e. versions >= 21.11

Reply:
Thanks for confirming, I was being vague because I wasn't sure exactly which version introduced the change 😅